src/styles/product_card.scss

```
@@ -0,0 +1,103 @@
+.product {
+  display: block;
+  width: 166px;
+  padding: 32px 16px 17px;
+  background-color: var(--white-color);
+  border: 1px solid var(--border-grey-color);
+  border-radius: 5px;
+  margin-right: 48px;
+  margin-bottom: 45px;
+    
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+}
+
+.product__picture {
+  box-sizing: border-box;
+  display: block;
+  width: 100%;
+  padding: 0 3px;
+  height: 134px;
+  justify-content: center;
+  margin-bottom: 40px;
+}
+
+.product__name {
+  display: block;
+  font-weight: 500;
+  font-size: 12px;
+  text-align: left;
+  color: var(--main-dark-color);
+  line-height: 18px;
+  margin-bottom: 4px;
+}
+
+.product__code {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  margin-bottom: 16px;
+  color: var( --ligth-gray-color);
+}
+
+.product__rate {
+  display: flex;
+  height: 16px;
+  justify-content: space-between;
+  width: 100%;
+
+  &-reviews {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    margin-top: 2px;
+    color: var(--main-dark-color);
+  }
+}
+
+.product__price {
+  display: flex;
+  justify-content: space-between;
+
+  &-amount {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    margin-top: 24px;
+    color: var(--main-dark-color);
+  }
+
+  &-text {
+    font-weight: 400;
+    font-size: 12px;
+    line-height: 18px;
+    color: var( --ligth-gray-color);
+    margin-top: 24px;
+  }
+}
+
+.button {
+  font-family: "Roboto", sans-serif;
+  background: var(--button-blue-color);
+  border-radius: 5px;
+  width: 100%;
+  height: 40px;
+  margin-top: 16px;
+  color: var(--white-color);
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  border: 1px solid var(--button-blue-color);
+  text-transform: uppercase;
+
+  &:hover {
+    border-radius: 5px;
+    background: var(--white-color);
+    color: var(--button-blue-color);
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    border-color: var(--button-blue-color);
```

Why do you repeat this style?
```suggestion
    background: var(--white-color);
    color: var(--button-blue-color);
```